### PR TITLE
feat: article content

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Closes https://app.clickup.com/t/8678v0jvy

Note: the content is hardcode, also im not handling external links or code hightlighting, this pr is mostly about applying the design styles

Pending stuff for other tasks:

- Parsing article markdown
- Handling external links
- Highlighting code?

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: public/images/quote.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="32" height="33" viewBox="0 0 32 33" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M0 13.796C0 9.08 3.836 5.258 8.571 5.258h.572a2.279 2.279 0 0 1 2.286 2.277 2.279 2.279 0 0 1-2.286 2.277H8.57c-2.207 0-4 1.786-4 3.984v.57h4.572c2.521 0 4.571 2.042 4.571 4.553v4.554c0 2.512-2.05 4.554-4.571 4.554H4.57c-2.52 0-4.57-2.042-4.57-4.554v-9.677Zm18.286 0c0-4.717 3.835-8.538 8.571-8.538h.572a2.279 2.279 0 0 1 2.285 2.277 2.279 2.279 0 0 1-2.285 2.277h-.572c-2.207 0-4 1.786-4 3.984v.57h4.572c2.521 0 4.571 2.042 4.571 4.553v4.554c0 2.512-2.05 4.554-4.571 4.554h-4.572c-2.521 0-4.571-2.042-4.571-4.554v-9.677Z" fill="#5060EE"/></svg>
```
Comment (ItsANameToo) [inline public/images/quote.svg:1]: remove the fill, better to set it later in the code so the icon is reusable

File: tailwind.config.js [hunk 2]
```diff
@@ -118,6 +118,225 @@ module.exports = {
             spacing: {
                 18: "4.5rem",
             },
+
+            typography: {
```
Comment (ItsANameToo) [inline tailwind.config.js:122]: will think about this setup

File: tailwind.config.js [hunk 3]
```diff
@@ -118,6 +118,225 @@ module.exports = {
...
+                        // "--tw-prose-counters": theme("colors.pink[600]"),
+                        // "--tw-prose-bullets": theme("colors.pink[400]"),
+                        "--tw-prose-hr": "rgb(var(--theme-color-secondary-300))",
+                        // "--tw-prose-quotes": theme("colors.pink[900]"),
+                        // "--tw-prose-quote-borders": theme("colors.pink[300]"),
+                        "--tw-prose-captions": "rgb(var(--theme-color-secondary-700))",
+                        "--tw-prose-code": "rgb(var(--theme-color-secondary-900))",
+                        "--tw-prose-pre-code": "rgb(var(--theme-color-secondary-900))",
+                        "--tw-prose-pre-bg": "rgb(var(--theme-color-secondary-50))",
+                        // "--tw-prose-th-borders": theme("colors.pink[300]"),
```
Comment (ItsANameToo) [inline tailwind.config.js:319]: can all the commented lines go? maybe link to the docs that outline them if it's for documentation purposes

File: resources/js/Pages/Articles/Components/ArticleContent.tsx [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+            itaque laboriosam magnam reiciendis atque, et molestias ex vitae ipsam commodi eligendi vero amet molestiae
+            rem.
+        </p>
+
+        <pre>
+            <code>
+                {`function greet() {
+    console.log("Hello, world!");
+}`}
+            </code>
```
Comment (ItsANameToo) [inline resources/js/Pages/Articles/Components/ArticleContent.tsx:186]: I think this is fine for now for code, it's not a coding blog

**Comment by alfonsobries** [comment]
@ItsANameToo updated to use a regular stylesheet

**Review by ItsANameToo — APPROVED** [review]

**Review by crnkovic — APPROVED** [review]

**Pull request merged**